lib/cc/cli/engines/validate.rb

```diff
@@ -0,0 +1,36 @@
+require "cc/analyzer"
+
+module CC
+  module CLI
+    module Engines
+      class Validate < EngineCommand
+        def run
+          if invalid_engines.any?
+            warn
+            true
```

Comment:
We would want this command to exit non-zero to signify failure in this case right? I'm not sure the mechanism commands have for controlling that, but if it's returning `true` here, that seems backwards.